roles/keycloak/templates/26.4/keycloak.conf.j2

```diff
@@ -0,0 +1,41 @@
+db=postgres
+db-username={{ keycloak_db.user }}
+db-password={{ keycloak_db.pass }}
+db-url=jdbc:postgresql://{{ keycloak_db_host }}/{{ keycloak_db.name }}
+
+
+# HTTP
+
+# The file path to a server certificate or certificate chain in PEM format.
+#https-certificate-file=${kc.home.dir}conf/server.crt.pem
+
+# The file path to a private key in PEM format.
+#https-certificate-key-file=${kc.home.dir}conf/server.key.pem
+
+# The proxy address forwarding mode if the server is behind a reverse proxy.
+#proxy=reencrypt
+#proxy=edge
```

Comment:
We can remove these two lines since the proxy option was deprecated in Keycloak v24 and removed in v26